lib/elixir/test/elixir/kernel/import_test.exs

```diff
@@ -137,6 +137,22 @@ defmodule Kernel.ImportTest do
     import String
   end
 
+  defmodule Sigils do
+    # when imported it should cause conflict
+    def is_integer(_), do: false
+
+    def sigil_X(_, _), do: :x
+
+    defmacro sigil_Y(_, _), do: :y
```

Comment:
Let's define a fake sigil too, `sigil__` and make sure it is not imported. You can check if something is imported by looking inside `__ENV__`.